pkg/common/si_helper.go

```diff
@@ -43,6 +43,10 @@ func createTagsForTask(pod *v1.Pod) map[string]string {
 	return tags
 }
 
+func CreateTagsForTask(pod *v1.Pod) map[string]string {
+	return createTagsForTask(pod)
+}
+
```

Comment:
Looks good, for simplicity, I think we can just rename this function to `CreateTagsForTask` to expose it.

Reply:
Sure, done.